app/forms/validators.py

```diff
@@ -424,3 +421,19 @@ def __call__(self, answer_values, is_mandatory):
                 self.messages["MANDATORY_QUESTION"], self.question_title
             )
             raise validators.ValidationError(message)
+
+
+def sanitise_mobile_number(data):
+    data = re.sub(r"[\s.,\t\-{}\[\]()/]", "", data)
+    return re.sub(r"^(\+044|044|0)", "", data)
```

Comment:
The `+44` example in the error message doesn't pass validation as we're not stripping `+44`here.

Reply:
I think `+044` was meant to be `+44` however, I don't think it's far-fetched to assume people do enter `+044` tbh.

Reply:
This was incorrect according to the card, so I've updated.